target/linux/ramips/image/mt7620.mk

```diff
@@ -61,13 +61,15 @@ TARGET_DEVICES += alfa-network_tube-e4g
 
 define Device/amit_jboot
   DLINK_IMAGE_OFFSET := 0x10000
-  KERNEL := $(KERNEL_DTB)
-  KERNEL_SIZE := 2048k
+  KERNEL := $(KERNEL_DTB) | uImage lzma -M 0x4f4b4c49
+  LOADER_FLASH_OFFS := 0x20000
+  LOADER_TYPE := bin
+  COMPILE := loader-$(1).bin
+  COMPILE/loader-$(1).bin := loader-okli-compile | pad-to 64k | lzma | pad-to 65480
```

Comment:
@mpratt14 , @adschm

Padding to `4040` instead `65480` wouldn't save us almost an eraseblock? The LZMA compressed `loader-$(1).bin` is less than 4040 bytes. For extra safety we could include `check-size 4040` as last step.
The Loader is able to find the `OKLI` magic anywhere in the flash. It just needs an exact `LOADER_FLASH_OFFS`.
Only rootfs had to be padded, IMHO.

Reply:
> Padding to `4040` instead `65480` wouldn't save us almost an eraseblock? 

I think it's better not to have an offset if its not necessary
probably not worth the effort
but if you do, then pad to 4k